flask-homework/app2.py

```diff
@@ -0,0 +1,184 @@
+from flask import Flask, request, render_template, redirect, abort
+import random
+
+app = Flask(__name__, template_folder='template_html')
+
+
+# Exercise 1
+@app.route('/users', methods=['GET'])
+def get_users():
+    names = ['Alice', 'Anna', 'Vitalij', 'Vlad', 'Bob']
+    random.shuffle(names)
+    return ', '.join(names)
+
+
+@app.route('/books', methods=['GET'])
+def get_books():
+    books = ['Kobzar', 'Harry Potter', 'Abetka', 'Witcher', 'Alice in Wonderland']
+    random.shuffle(books)
+    html_list = '<ul>'
+    for book in books:
+        html_list += f'<li>{book}</li>'
+    html_list += '</ul>'
+    return html_list
+
+
+# Exercise 2
+@app.route('/users/id', methods=['GET'])
```

Comment:
it should be <id> parameter
```suggestion
@app.get('/users/<id>')
def get_users_by_id(id):
```